src/mysql.go

```diff
@@ -62,7 +63,13 @@ func main() {
 
 	log.SetupLogging(args.Verbose)
 
-	e := i.LocalEntity()
+	var e *integration.Entity
+	if args.RemoteMonitoring {
+		e, err = i.Entity(fmt.Sprint(args.Hostname, ":", args.Port), "mysql")
```

Comment:
I would change `mysql` argument and replace it with the entity type, in this case I suppose it's `instance`, @ikavanaghNR WDYT?

In the [json schema for protocol v2](https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/v2tov3.md#json-v2-schema-changes) you'll see that we send the associated entity, and the entity type attribute of such entity comes from the `namespace` attribute of [`EntityMetadata`](https://github.com/newrelic/infra-integrations-sdk/blob/2701a5446f2f412eb940c9cc2d0e778decd782e7/integration/entity.go#L29), which in turn is taken from the second parameter of [`Entity`](https://github.com/newrelic/infra-integrations-sdk/blob/2701a5446f2f412eb940c9cc2d0e778decd782e7/integration/integration.go#L121).

Also this is the approach taken by BM; for example, [MSSQL instace](https://github.com/newrelic/nri-mssql/blob/2c0a8b1bf32a73202e277e029ca0aae1ba12c6d8/src/instance/sql_instance.go#L30).